### PR TITLE
Added example, explanation and solution to "Use Hex Code to Mix Colors"

### PR DESCRIPTION
**Pull request by KevTrang00**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@KevinatorTrainer5 Please remove the problem and objective parts here.  The user can always look at the original challenge to see this content.

Thank you.

**Comment by thecodingaviator** [comment]
@RandellDawson I really like this PR, but since we haven't heard from the OP in some time, I'm making changes on their behalf

**Comment by RandellDawson** [comment]
@thecodingaviator 
The following seems repetitive.
````
### After changing `color` of other classes
```html
<style>
  .green-text {
    color: #00FF00; 
}
  .dodger-blue-text {
    color: #1E90FF; 
} 
  .orange-text {
    color: #FFA500; 
}  
</style>
```

## Final Solution
Hence, the final solution reveals that we have changed the `color` element of these text classes with hex codes shown below:
```html
<style>
  .red-text {
    color: #FF0000;
  }
  .green-text {
    color: #00FF00;
  }
  .dodger-blue-text {
    color: #1E90FF;
  }
  .orange-text {
    color: #FFA500;
  }
</style>
````
Maybe combine the two like:
````
## Final Solution
After changing `color` of other classes, the final solution reveals that we have changed the `color` element of these text classes with hex codes shown below:
```html
<style>
  .red-text {
    color: #FF0000;
  }
  .green-text {
    color: #00FF00;
  }
  .dodger-blue-text {
    color: #1E90FF;
  }
  .orange-text {
    color: #FFA500;
  }
</style>
````

**Comment by thecodingaviator** [comment]
@RandellDawson looks better?

File: guide/english/certifications/responsive-web-design/basic-css/use-hex-code-to-mix-colors/index.md [hunk 1]
```diff
@@ -3,8 +3,64 @@ title: Use Hex Code to Mix Colors
...
+### After changing `color` of class `.red-text`
+```html
+<style>
+.red-text {
+  color: #FF0000; /* Color has changed into red when we used this hex code from the table*/
+}
+</style>
+```
+
+## Final Solution
```
Comment (RandellDawson) [inline guide/english/certifications/responsive-web-design/basic-css/use-hex-code-to-mix-colors/index.md]:
```suggestion
### Final Solution
```

Comment (thecodingaviator) [reply guide/english/certifications/responsive-web-design/basic-css/use-hex-code-to-mix-colors/index.md]: done

**Review by RandellDawson — APPROVED** [review]
LGTM.  Merge when build finishes.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍 

@RandellDawson Roger

**Comment by thecodingaviator** [comment]
@KevinatorTrainer5 Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**